libstb/trustedboot.c

```diff
@@ -38,14 +38,17 @@ static bool boot_services_exited = false;
  * mapped according to the TCG PC Client Platform Firmware Profile
  * specification, Revision 00.21
  * Only resources included in this whitelist can be measured.
+ *
+ * NOTE: modified to suit Heads use case and confine all skiboot PCR updates
```

Comment:
Let's drop this whole commit. PCR usage and capping performed by Skiboot is compliant with https://tianocore-docs.github.io/edk2-TrustedBootChain/release-1.00/edk2-TrustedBootChain-release-1.00.pdf, and trying to change it would most likely result in having to maintain our fork of Skiboot indefinitely.

PCR usage that Heads is used to comes from strange assumptions and design decisions done by vboot, like proprietary event log format, mixed use of PCR banks or definitions of DRTM/SRTM that are just plain wrong. It shouldn't matter that PCR4 is extended from non-zeroed state, all measurements performed should be recorded in the event log and can be replayed from there.